EditorsDraft/index.html

```diff
@@ -2336,55 +2546,163 @@
+
+These new requirements might result in:
+
+* submission of proposals to the OpenActive Community Group for revisions to this 
+specification
+* creation of new specifications that support the needs fo specific types of publisher. 
+For example a group of publishers might wish to define a detailed specification for 
+describing the characteristics of cycle tracks
+* creation of bespoke properties that support the needs of a specific publisher 
+and their data users
+
+In all of these cases we encourage discussion of requirements and extensions via 
+the OpenActive Community Group, as the primary forum for standarding the 
+publication of opportunity data.
+
+#### Defining and Using Custom Namespaces
+
+The data model described in this specification is defined in 
+the [OpenActive JSON-LD context](https://www.openactive.io/ns/oa.jsonld) defined by [[OpenActive-Vocabulary]].
+
+They <em class="rfc2119">SHOULD</em> also:
+
+* publish public, human-readable documentation that describes their extension, 
+including any custom properties, types and controlled vocabularies that are 
+being used
+* share the documentation with the OpenActive community group
+* look for opportunities to align their data model with revisions to the  
+standard and other extensions defined in the community
+
+Publish <em class="rfc2119">MUST</em> also
+
+* avoid using names for custom types or properties that clash with the core 
+data model
+* publish a custom [JSON-LD context](https://www.w3.org/TR/json-ld/#the-context) to provide a machine-readable version of their 
+extension 
+* include a reference to their context in their published data
+
+A full tutorial on creating JSON-LD contexts is outside the scope of this specification. 
+
+The following example shows a simple custom JSON-LD context that defines a new namespace
+`ext` and three custom properties.
+
+<pre class="example" title="Example of a custom context"><code>
+{
+   "@context":{
+      "label":"http://www.w3.org/2000/01/rdf-schema#label",
+      "xsd":"http://www.w3.org/2001/XMLSchema#",
+      "ext":"http://example.org/jsonld#",
+      "id":"@id"
+   },
+   "ext:myCustomProperty":{
+      "id":"ext:myCustomProperty",
+      "label":"This is the label"
+   },
+   "ext:myNumericProperty":{
+      "id":"ext:myNumericProperty",
+      "label":"This is a property with a number value",
+      "@type":"xsd:integer"
+   },
+   "ext:myURLProperty":{
+      "id":"ext:myURLProperty",
+      "label":"This is a property with a URI value",
+      "@type":"@id"
+   }
+}
+</code></pre>
+
+[[JSON-LD]] allows data to be [published with reference to multiple contexts](https://www.w3.org/TR/json-ld/#advanced-context-usage). 
+
+Assuming that the previous JSON-LD context was published to `https://example.org/custom.jsonld`, then 
+the following example shows how to use this extension:
+
+<pre class="example" title="Example of a custom context"><code>
+{
+  "@context": [
+    "https://www.openactive.io/ns/oa.jsonld",
```

Comment:
Note - the validator currently fails for arrays in context

Reply:
Not any more..!